official/resnet/imagenet_main.py

```diff
@@ -281,9 +281,16 @@ def imagenet_model_fn(features, labels, mode, params):
                                          multi_gpu=params['multi_gpu'])
 
 
-def main(argv):
+def main(argv=None):
+  argv = sys.argv if argv is None else argv
```

Comment:
Do we still need this at all if we're calling main directly?

Reply:
The end to end test passes its own args instead of using sys.argv.

Reply:
I think it's more conventional then to have the `if name ==` section pass in sys.argv, rather than having this extra line here.